### PR TITLE
Add mutation test coverage

### PR DESCRIPTION
**Pull request by sivas23**
**Before creating a pull request make sure that:**

### JIRA link (if applicable) ###
not raised ticket for this

### Change description ###
added mutation test coverage

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: service/build.gradle [hunk 1]
```diff
@@ -125,9 +126,16 @@ sonarqube {
     property "sonar.projectName", "FPL :: Service"
     property "sonar.jacoco.reportPath", file("${project.buildDir}/jacoco/test.exec")
     property "sonar.jacoco.itReportPath", file("${project.buildDir}/jacoco/integration.exec")
+    property "sonar.host.url", "https://sonar.reform.hmcts.net/"
```
Comment (geek-soft) [inline service/build.gradle]: Remove, we use sonar cloud now.

File: service/build.gradle [hunk 2]
```diff
@@ -125,9 +126,16 @@ sonarqube {
...
+    //property "sonar.host.url", "http://localhost:9000/"
```
Comment (geek-soft) [inline service/build.gradle]: Remove commented out code.

File: service/build.gradle [hunk 3]
```diff
@@ -158,6 +166,9 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'pdf-generator', version: '1.0.2'
 
   compile group: 'commons-io', name: 'commons-io', version: '2.6'
+  compile group: 'org.pitest', name: 'pitest', version: '1.3.2'
+  compile group:'info.solidsoft.gradle.pitest', name: 'gradle-pitest-plugin', version: '1.3.0'
```
Comment (geek-soft) [inline service/build.gradle]: Plugins should be declared in plugins section. Move it.

Comment (sivas23) [reply service/build.gradle]: These are dependencies.

File: service/build.gradle (hunk 2 above)
Comment (sivas23) [reply service/build.gradle]: Removed commented lines

File: service/build.gradle (hunk 1 above)
Comment (sivas23) [reply service/build.gradle]: Though we are use sonar cloud we use https://sonar.reform.hmcts.net/ for sonar.host.url which in turn cloud-native platform team pointing to sonar cloud https://sonarcloud.io

Comment (geek-soft) [reply service/build.gradle]: Remove `sonar.host.url` property.

File: service/build.gradle [hunk 4]
```diff
@@ -125,9 +126,14 @@ sonarqube {
...
     property "sonar.jacoco.reportPath", file("${project.buildDir}/jacoco/test.exec")
     property "sonar.jacoco.itReportPath", file("${project.buildDir}/jacoco/integration.exec")
+    property "sonar.host.url", "https://sonar.reform.hmcts.net/"
+    property "sonar.pitest.mode", "reuseReport"
+    property "sonar.pitest.reportsDirectory", "/build/reports/pitest"
   }
 }
 
+
+
```
Comment (geek-soft) [inline service/build.gradle]: Remove excessive lines

File: service/build.gradle (hunk 3 above)
Comment (geek-soft) [reply service/build.gradle]: Looks like a plugin.

**Pull request merged**